### PR TITLE
MHV-55181 copies mhv/shared folder to platform

### PR DESCRIPTION
**Pull request by KolbyVA**
## Summary
Copies the mhv/shared folder into platform for importing using `@department-of-veterans-affairs`. 
Also updates import in Secure-messaging's App.jsx to ensure imports are working.

## Related issue(s)
https://jira.devops.va.gov/browse/MHV-55181
> Isolated Apps - Copy MHV Shared folder content over to platform site

## Testing done
Manually tested locally

## Screenshots
No UI change

## What areas of the site does it impact?
platform/mhv
secure-messaging

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by mmoyer-va — DISMISSED** [review]

**Review by alex-morgun-1 — DISMISSED** [review]
Approved pending review by @mattwrightva of that codeql check

**Review by wright0776 — APPROVED** [review]
LGTM!

File: src/platform/mhv/hooks/usePrintTitle.jsx
Comment (wright0776) [reply src/platform/mhv/hooks/usePrintTitle.jsx:14]: Feel free to rewrite this without the replace function :)

**Review by alex-morgun-1 — APPROVED** [review]
Great work 👍🏻 no concerns from SM perspective. Will monitor datadogRum hook in staging

**Review by Andrew565 — APPROVED** [review]

**Pull request merged**
